ceph-salt-formula/salt/_modules/ceph_orch.py

```diff
@@ -2,14 +2,11 @@
 import json
 
 def configured():
-    ret = __salt__['cmd.run_all']("sh -c 'type ceph'")
-    if ret['retcode'] != 0:
-        return False
     if not __salt__['file.file_exists']("/etc/ceph/ceph.conf"):
         return False
     if not __salt__['file.file_exists']("/etc/ceph/ceph.client.admin.keyring"):
         return False
-    ret = __salt__['cmd.run_all']("ceph orch status")
+    ret = __salt__['cmd.run_all']("timeout 15 ceph orch status")
```

Comment:
could we double the timeout value to 30 seconds? 15 seconds makes me nervous

Reply:
Done